libs/common/src/main/kotlin/soeknad/dataklasser/common/Personer.kt

```diff
@@ -80,6 +79,28 @@ data class Barn(
     override val type = PersonType.BARN
 }
 
+data class Gjenlevende(
```

Comment:
Kanskje du kan lage en oppgave som går på å bruke https://github.com/navikt/pensjon-etterlatte-libs/tree/main/common/src/main/kotlin/no/nav/etterlatte/libs/common/innsendtsoeknad for disse tingene? Slik at dette blir delt på tvers av selvbetjening og sak?